BugsnagPerformance/Assets/BugsnagPerformance/Scripts/Public/PerformanceConfiguration.cs

```diff
@@ -67,5 +70,14 @@ internal List<Func<Span, bool>> GetOnSpanEndCallbacks()
 
         internal bool IsFixedSamplingProbability => SamplingProbability >= 0;
 
+        public string GetEndpoint()
+        {
+            if(string.IsNullOrEmpty(Endpoint) || Endpoint == LEGACY_DEFAULT_ENDPOINT)

```

Comment:
I don't see a need to catch the case where the developer has set the config to the original SaaS endpoint. In fact that could get confusing if for some reason we want them to use the non-API key one.

I suggest we just do the simple check for null/empty and not compare with the legacy.

Reply:
this is because if someone was using the bugsnag config window and the default endpoint and then updates to the new release, then it will be read from the config file anyway and override the new default.